src/libs/ajax/data-table-providers/WdsDataTableProvider.test.ts

```diff
@@ -138,22 +146,28 @@ describe('WdsDataTableProvider', () => {
     return Promise.resolve(testProxyUrlResponse)
   }
 
+  const createAppV2MockImpl: AppsContract['createAppV2'] = (_workspaceId: string) => {
+    return Promise.resolve(testProxyUrlResponse)
```

Comment:
this isn't right - the `createAppV2` endpoint returns a 202 with no response body, right? (that's from memory, I haven't tried it in practice). Here, the mock is returning the same body as the `listAppsV2` endpoint.

That said, I don't think we ever use the response anywhere so it's not a big deal. how about 
```
return Promise.resolve('')
```
instead?

Reply:
Perhaps I should've noted here prior, but this was necessary in order for the mock to successfully work in the logic within the unit tests.

Nevertheless, changed the return value here!

```
jest.mock('./WdsDataTableProvider', () => ({ createLeoAppWithErrorHandling: () => {} }))
```